client.js

```diff
@@ -121,27 +121,36 @@ RingpopClient.prototype._request = function _request(opts, endpoint, head, body,
         return;
     }
 
-    this.subChannel.waitForIdentified({
-        host: opts.host
-    }, function onIdentified(err) {
-        if (err) {
-            callback(err);
-            return;
-        }
+    try {
+        request();
+    } catch(e) {
+        callback(e);
+        return;
+    }
```

Comment:
Crashes are bugs. File bug reports on tchannel.

Don't just swallow and hide bugs.

We continue on uncaught exceptions already so no need for this kind of defensive programming